scripts/base/init-bare.zeek

```diff
@@ -4436,8 +4436,48 @@ type ModbusHeaders: record {
 	uid:           count;
 	## MODBUS function code
 	function_code: count;
+	## Full length of modbus PDU in bytes
+	pdu_length:    count;
```

Comment:
The length field in the header is the length of the data immediately following that field. The requested PDU length field is basically the length of everything after that header, which is a single byte field. I could see just removing the `pdu_length` field altogether (reverting that commit) and telling the user to just calculate it themselves from the `len` field in the header.

Reply:
> and telling the user to just calculate it themselves from the len field in the header.

That's what I meant, yes, include the original `len` rather than the derived field (still need to extend `ModbusHeaders` by `len`).

Reply:
Gotcha, implemented that.